index.html

```diff
@@ -189,10 +182,100 @@ <h3>per month</h3></div>
                     <button class="portfolio-button--color">Sign Up</button>
                 </div>
             </div>
+        </div>
+    </section>
 
-
+    <section class="contact-me">
+        <div class="contact-items mg-btm">
+            <h2 class="mg-btm">Contact Me</h2>
+            <div class="block-wrapper">
+                <a class="block" href="mailto:email@email.com">
+                    <div class="icon-message">
+                        <i class="fa fa-envelope mg-btm"></i>
+                        <p>email@email.com</p>
+                    </div>
+                </a>
+                <a class="block" href="https://maps.google.com/?q=50.45112524552318,30.526240341027922">
+                    <div class="icon-message">
+                        <i class="fa fa-map-marker mg-btm"></i>
+                        <p>Kyiv, UA</p>
+                    </div>
+                </a>
+                <a class="block" href="tel:068-400-8020">
+                    <div class="icon-message">
+                        <i class="fa fa-phone mg-btm"></i>
+                        <p>0684008020</p>
+                    </div>
+                </a>
+            </div>
+        </div>
+        <div class="form-wrapper">
+            <form id="form1">
+                <label for="name">Name</label>
+                <input type="text" id="name" placeholder="John Doe">
+                <label for="email">Email</label>
+                <input type="text" id="email" placeholder="john@example.com">
```

Comment:
if `type="email"` when you submit the form with the wrong email, you'll see validation:
<img width="517" alt="Screenshot 2021-12-06 at 12 37 49" src="https://user-images.githubusercontent.com/26774793/144831954-1d31f754-e2be-421a-ad4a-33009335e1fd.png">